WordPress/Classes/Utility/Spotlight/Spotlightable.swift

```diff
@@ -23,6 +23,28 @@ class SpotlightableButton: UIButton, Spotlightable {
         }
     }
 
+    func startLoading() {
+        setTitle("", for: .normal)
+        activityIndicator.startAnimating()
+    }
+
+    func stopLoading() {
+        activityIndicator.stopAnimating()
```

Comment:
Should we restore the title when we stop loading?

Reply:
Done in 1e49f70d7941a7b87ab0c07795ec71643224fcac